.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.5.3
```

Comment:
So we want to make sure we stay with 2.4.1 since using 2.5.3 will break the Travis CI build.

Reply:
Ah nevermind. We should be on 2.5.3